packages/apify/src/utils_request.ts

```diff
@@ -243,7 +243,8 @@ export async function requestAsBrowser<T = string>(options: RequestAsBrowserOpti
                     return reject(e);
                 }
 
-                addResponsePropertiesToStream(duplexStream, res);
+                // TODO: stream vs response interface?
+                addResponsePropertiesToStream(duplexStream as any, res);
```

Comment:
This seems to be a merge/rebase mistake, as in the branch this cast isn't needed